### ISSUE TITLE
consolidate demo/README.md and examples/zone-printer/README.md

### ISSUE THREAD

**Issue by ahmetb** [GoogleCloudPlatform/k8s-multicluster-ingress#161]
I think you should move zone-printer/README.md to demo/.

We can easily change zone-printer with a different app that no longer uses nginx+configmap and the demo/sample/example isn't much about the app itself, but rather instructions.

**Comment by ahmetb** [comment]
I understand `demo/` is for bash prompt automation. Consider moving it out of project root. It's not an important component and users will not use it.

**Comment by nikhiljindal** [comment]
Yes `demo/` has scripts that people can use to demo kubemci. We could move it out of the repo, it is there for discoverability.

I dont see this as an important issue.
